templates/blog/article.html

```diff
@@ -32,111 +34,240 @@
+            <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c" id="Facebook_Click_ID__c" value="" />
+            <input type="hidden" name="thankyoumessage" value="Thank you for subscribing!<br />You will begin receiving emails as new content is posted. You may unsubscribe any time by clicking the link in the email.">
+        </form>
+      </div>
+      <div class="l-article__main p-section">
+        <div class="u-text-max-width">
+            {{ article.content.rendered|safe }}
         </div>
       </div>
+    </section>
 
-      <div class="col-4">
-        <div style="margin-top:1rem;">
-          {% include 'blog/newsletter-form.html' %}
+    {% if related_articles %}
+    <section class="p-section">
+      <div class="u-fixed-width">
+        <h4 class="p-muted-heading">Related posts</h4>
+      </div>
+    
+      {% for related_article in related_articles %}
+      <hr class="p-rule is-fixed-width">
+      <div class="p-block row">
+        <div class="col-3 col-medium-2">
+          <p><a href="/blog/author/{{related_article.author.slug}}">{{ related_article.author.name }}</a>
+            <br>{{ related_article.date }}
+          </p>
         </div>
+    
+        <div class="col-6 col-medium-4">
+          <div class="p-block">
+            <h3 class="p-heading--2">
+              <a href="/blog/{{ related_article.slug }}">{{ related_article.title.rendered|safe }}</a>
+            </h3>
+          </div>
+          <p class="u-hide--small">
+            {{ related_article.excerpt.raw.replace("[…]", "")|truncate(1000) }}...
+          </p>
+    
+          <span class="p-chip--information">
+            <span class="p-chip__value">
+            {% if request.path == "topic/blog/cloud-and-server" %}
+                  Cloud and Server
+              {% elif request.path == "topic/blog/desktop" %}
+                  Desktop
+              {% elif request.path == "topic/blog/internet-of-things" %}
+                  Internet of Things
+              {% else %}
+                {% if related_article.group %}
+                  {{ related_article.group.name }}
+                {% else %}
+                  Ubuntu
+                {% endif %}
+              {% endif %}
+            </span>
+          </span>
+            <span class="p-chip--information">
+              <span class="p-chip__value">
+                {% include 'blog/singular-category.html' %}
+              </span>
+            </span>
+          </div>
+    
+          <div class="col-2 col-start-large-11 u-hide--medium u-hide--small">
+            {% if related_article.image and related_article.image.source_url %}
+            <div class="u-aspect-ratio--3-2">
+              <a href="/blog/{{ article.slug }}" aria-hidden="true" tabindex="-1">
+                <div class="p-image-wrapper">{{ related_article.image.rendered|safe }}</div>
```

Comment:
What is `p-image-wrapper`

Reply:
adds .5rem spacing so images don't touch p-rules positioned above.